multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -710,6 +711,9 @@ def fit(
             label_columns=self._label_column,
             provided_column_types=column_types,
         )
+        if self._problem_type == NER:
+            column_types = infer_ner_column_type(column_types=column_types)
```

Comment:
Can we make this problem_type agnostic ?

Reply:
Good suggestion! Moved `infer_ner_column_type ` into `infer_column_types`.